README.md

```diff
@@ -197,6 +197,43 @@ ReactDOM.render(
 There is an example app with CRA, Material-UI and Storybook Addon [Demo](https://react-theming.github.io/theming-material-ui/) [Source](https://github.com/react-theming/theming-material-ui)
 
 
+## Use your output of the selected value
```

Comment:
Thanks for adding documentation for the new feature. Let's put it to the section where all features are described - I think next to the "background colors" would be good

Reply:
Done